plugins/all.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: all
+spec:
+  homepage: https://github.com/onatm/kall
+  shortDescription: Run 'kubectl get' on all the contexts found in your .kube/config
+  version: "v1.0.0"
+  description: |
+    This command allows you to run get command on multiple contexts' resources.
```

Comment:
Why only `get`? I imagine that all commands could also be useful.

Reply:
I thought making it `get` only would be safer but I believe explicitly calling `all` command already states that commands would run for all contexts. I'll create a new release to make it work for all other commands.